### PR TITLE
[Merged by Bors] - chore: bump dependencies

### PR DESCRIPTION
**Pull request by kim-em**

### CODE REVIEW

File: Mathlib/Init/Data/Rat/Basic.lean [hunk 1]
```diff
@@ -11,7 +11,5 @@ import Mathlib.Mathport.Rename
 # Align statements for declarations from Std
 -/
 
-#align rat.ext Rat.ext
 #align rat Rat
-#align rat.ext_iff Rat.ext_iff
```
Comment (Ruben-VandeVelde) [inline Mathlib/Init/Data/Rat/Basic.lean:16]: Why is this needed?

Comment (semorrison) [reply Mathlib/Init/Data/Rat/Basic.lean:16]: The `@[ext]` attribute was removed in Std, so these don't exist. We should change to #noalign, however.

Comment (semorrison) [reply Mathlib/Init/Data/Rat/Basic.lean:16]: Hmm, but that can't be right, because `Rat.ext` exists later.

Comment (semorrison) [reply Mathlib/Init/Data/Rat/Basic.lean:16]: Okay, sorted. Rat.ext still exists, in a differnet file in Std, ext_iff has been removed.

Comment (eric-wieser) [reply Mathlib/Init/Data/Rat/Basic.lean:16]: Surely we should keep `ext_iff`?
